backend/src/test/java/springweb/backend/ItemControllerIntegrationTest.java

```diff
@@ -35,15 +35,15 @@ class ItemControllerIntegrationTest {
     MockRestServiceServer mockRestServiceServer;
 
     @Test
-    void getAllItems()  throws Exception {
+    void getAllItems() throws Exception {
```

Comment:
May be you should choose the test name similar to other tests. Like "getAllItems_shouldReturnEmptyListWhenNoItemsExist"